### PR TITLE
Reduce toilet volume

### PR DESCRIPTION
**Pull request by HaasJona**
Reduce volume of toilet sound so that it doesn't drown ride music and other sounds. #4519 

This is for discussion / testing.

### CODE REVIEW

File: src/audio/audio.c [hunk 1]
```diff
@@ -234,6 +234,8 @@ rct_audio_params audio_get_params_from_location(int soundId, const rct_xyz16 *lo
 		params.pan = viewport->x + (vx >> viewport->zoom);
 		params.volume = _volumeAdjust[soundId] + ((-1024 * viewport->zoom - 1) << volumeDown) + 1;
 
+		printf("%d\n", params.volume);
```
Comment (janisozaur) [inline src/audio/audio.c]: remove this

Comment (AaronVanGeffen) [inline src/audio/audio.c]: Leftover `printf`.

Comment (janisozaur) [inline src/audio/audio.c]: first :P

Comment (HaasJona) [inline src/audio/audio.c]: Fixed. Sorry, I'm tired.

**Pull request merged**